README.rst

```diff
@@ -94,8 +102,17 @@ Raw HTML    ``extra.html('<div>Additional HTML</div>')``
 Plain text  ``extra.text('Add some simple Text')``
 URL         ``extra.url('http://www.example.com/')``
 Image       ``extra.image(image, mime_type='image/gif', extension='gif')``
+Image       ``extra.image('/path/to/file.png')``
+Image       ``extra.image('http://some_image.png')``
 ==========  ============================================
 
+**Note**: When adding an image from file, the path can be either absolute
+or relative.
+
+**Note**: When using ``--self-contained-html``, images added as files or links
+may not work as expected, see the
+:ref:`Creating a self-contained report <self-contained>` section for more info.
```

Comment:
You don't need to create a reference as all titles can already be referenced. This would work as:

```rst
**Note**: When using ``--self-contained-html``, images added as files or links
may not work as expected, see section `Creating a self-contained report`_ for
more info.
```